src/commands/install.ts

```diff
@@ -335,7 +349,12 @@ export async function install(
       }),
       rollupPluginCss(),
       rollupPluginCommonjs({
-        extensions: ['.js', '.cjs'], // Default: [ '.js' ]
+        extensions: ['.js', '.cjs'],
+        // Workaround: CJS -> ESM isn't supported yet by the plugin, so we needed
+        // to add our own custom workaround here. Requires a fork of
+        // rollupPluginCommonjs that supports the "externalEsm" option.
+        // @ts-ignore
+        externalEsm: filterExternalPackagesForEsm(externalPackages),
```

Comment:
Where is this forked?

Reply:
Forked in your own project setup, via an npm/yarn alias. The fork I've written is internal only for now